packages/api/tests/es/test-stats.js

```diff
@@ -125,3 +125,80 @@ test.serial('Count returns correct granule and collection count', async (t) => {
 
   t.is(collectionCountResult.meta.count, 1);
 });
+
+test.serial('Count returns correct count for date range', async (t) => {
+  await Promise.all([
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2({
+      updatedAt: new Date(2020, 0, 27)
+    }), t.context.esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2(), t.context.esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2({ status: 'failed' }), t.context.esAlias),
+    indexer.indexGranule(esClient, fakeGranuleFactoryV2({ status: 'failed', updatedAt: new Date(2020, 0, 29) }), t.context.esAlias)
+  ]);
+
+  let stats = new Stats(
+    { queryStringParameters: { updatedAt__to: (new Date(2020, 0, 30)).getTime() } },
```

Comment:
should these be `timestamp__to` instead of updatedAt__to? Or I suppose updatedAt is also a valid ES field. so It probably doesn't matter.